src/main/java/ipman/Storage.java

```
@@ -0,0 +1,81 @@
+package ipman;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Handles storage operations for tasks.
+ * This class provides methods to read from and write to a file.
+ * It supports both temporary and specified file storage.
+ *
+ * @author miloaisdino
+ */
+public class Storage {
+    private final File file;
+
+    /**
+     * Constructs a Storage instance using a temporary file.
+     * The temporary file is created with a unique name based on the current timestamp.
+     */
+    public Storage() {
+        file = new File(String.valueOf("tmp_" + System.currentTimeMillis() + ".txt"));
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            //do nothing
```

Empty catch block should be avoided